src/Enzyme.jl

```diff
@@ -798,18 +779,16 @@ of this is the transpose of the Forward [`jacobian`](@ref)
 Example:
 
 ```jldoctest
-using Enzyme
-
 f(x) = [x[1]*x[2], x[2]]
 
 grad = jacobian(Reverse, f, [2.0, 3.0])
 
 # output
 
-(([3.0, 2.0], [0.0, 1.0]),)
+([3.0, 2.0],)
```

Comment:
@wsmoses that this didn't work worries me. Should this be `jacobian(Reverse, f, [2.0, 3.0]; n_outs=Val(2))`

Reply:
It should absolutely be n_outs=2 and that should also absolutely NOT have a default value. It needs to be precisely the length of the returned array (here 2).

Reply:
We will need to present a better error message to users then. Keyword values are supposed to have a default value

Reply:
Make it a required arg? I just was worried that having two args would be confusing/lead to mis ordering